.github/workflows/on-pull-request.yml

```diff
@@ -0,0 +1,15 @@
+# Pull request workflow to run CI on PRs.
+# Triggered on opened or reopened pull requests.
+# Defines one job that reuses the ci.yml workflow.
+# ci.yml will run lint, tests, coverage, build for the PR.
+
+name: CI (on pull-request)
+run-name: PR from branch ${{ github.ref }}
+
+# Will run CI workflow on an opened or reopened PR for easy reference. For compilation, unit tests, code coverage etc.
+on:
+  pull_request:
+    types: [opened, reopened]
+jobs:
+  Continuous-Integration:
+    uses: ./.github/workflows/ci.yml
```

Comment:
Add name: on pull request. This is required to set it as a required status check
see: https://github.com/SeattleColleges/nsc-events-nextjs/blob/main/.github/workflows/on-pull-request.yml